backend/src/test/java/com/pickpick/channel/application/ChannelServiceTest.java

```diff
@@ -76,6 +95,55 @@ void findAll() {
         );
     }
 
+    @DisplayName("전체 채널을 조회하면 사용자가 입장한 채널만 조회된다")
+    @Test
+    @Transactional
+    void findChannelsHasUser() throws SlackApiException, IOException {
+        // given
+        Workspace workspace = workspaces.save(new Workspace("t12345", "xoxb-token-1234"));
+        Member yeonLog = saveMember(workspace);
+
+        Channel notice = channels.save(NOTICE.create(workspace));
+        Channel freeChat = channels.save(FREE_CHAT.create(workspace));
+        Channel qna = channels.save(QNA.create(workspace));
+
+        channelSubscriptions.save(new ChannelSubscription(freeChat, yeonLog, 1));
+
+        given(methodsClient.conversationsList((RequestConfigurator<ConversationsListRequestBuilder>) any()))
+                .willReturn(generateConversationsListResponse(notice));
+
+        // when
+        List<String> channelNames = channelService.findByWorkspace(yeonLog.getId())
+                .getChannels()
+                .stream()
+                .map(ChannelResponse::getName)
+                .collect(Collectors.toList());
+
+        // then
+        assertThat(channelNames).isNotEmpty();
+        assertThat(channelNames).doesNotContain(freeChat.getName(), qna.getName());
+    }
+
+    private Member saveMember(final Workspace workspace) {
+        Member member = new Member("U00001", "연로그", "https://yeonLog.png", workspace);
+        member.markLoggedIn("xoxp-token");
+        return members.save(member);
+    }
+
+    private ConversationsListResponse generateConversationsListResponse(final Channel... channels) {
+        ConversationsListResponse conversationsListResponse = new ConversationsListResponse();
+        conversationsListResponse.setOk(true);
+        List<Conversation> conversations = new ArrayList<>();
+        for (Channel channel : channels) {
+            Conversation conversation = new Conversation();
+            conversation.setId(channel.getSlackId());
+            conversation.setMember(true);
+            conversations.add(conversation);
+        }
```

Comment:
```suggestion
        List<Conversation> conversations = Arrays.stream(channels)
                .map(channel -> Conversation.builder()
                        .id(channel.getSlackId())
                        .isMember(true).build())
                .collect(Collectors.toList());
```
빌더와 스트림 사용 어떨까요? 자동 setter라서 그런지 setIsMember가 아니라 setMember라서 처음 딱 보고는 `왜 true지?` 라고 생각이 들었어서 제안해봅니다!

Reply:
```
참고 사항

테스트는 일단 돌아만 가게 짜놨습니다!
피드백 주시면 반영하고 그냥 두시면 테스트 전체 리팩터링 때 다루게 될 것 같아요~
``` 
이걸 나중에 봤네요 ㅠㅠ